cloud/gce/network.tf

```diff
@@ -19,7 +19,9 @@ resource "google_compute_target_pool" "default" {
 resource "google_compute_forwarding_rule" "default" {
```

Comment:
I think almost all the `network.tf` stuff can go away, though we may still need the firewall rule.